docs/update-gh-pages.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# This script updates the documentation on knora.org by pushing a commit to the gh-pages branch
+# of the Knora repository on GitHub.
+
+# Before you run this script, you must be on the develop branch, having committed any changes
+# you made there.
+
+# If you don't want git to ask you for your username and password, use SSH instad of HTTPS
+# to clone the Knora repository.
+
+TMP_HTML="/tmp/knora-html"
+
+set -e
+
+# Build the HTML docs.
+
+make html
+make jsonformat
+
+# Copy the built HTML docs to a temporary directory.
+
+rm -rf $TMP_HTML
+mkdir $TMP_HTML
+cp -R _build/html $TMP_HTML/manual
+cp -R _format_docu $TMP_HTML/api
+
+# Switch to the gh-pages branch and remove the existing HTML docs from it.
+
+git checkout gh-pages
+rm -rf ../documentation/manual
```

Comment:
You are just deleting all the files. However, they remain under version control appearing as missing in git. I think you should use `git rm`. Imagine the case that someone removes a file from the docs. It will appear as missing in `gh-pages` although it is not longer under version control in the `docs` folder.